server/etc/pulp/server.conf

```diff
@@ -164,7 +163,6 @@
 
 [data_reaping]
 # reaper_interval: 0.25
-# archived_calls: 0.5
```

Comment:
You're saying the other ones marked as deprecated are still used while this one is not. Removing this line from the `server.conf` and adding the release note only sounds fine to me. Also the release note needs to mention that both the setting is removed from server.conf and that the `archived_calls` collection will be removed with a migration.